src/invwarpedview.jl

```diff
@@ -0,0 +1,118 @@
+"""
+    InvWarpedView(img, tinv, [indices]) -> wv
+
+Create a view of `img` that lazily transforms any given index `I`
+passed to `wv[I]` to correspond to `img[inv(tinv)(I)]`. While
+technically this approach is known as backward mode warping,
+`InvWarpedView` is created using the forward transformation.
```

Comment:
Perhaps "note that `InvWarpedView` is created by supplying the forward transformation"?

I wonder if it's a little confusing to call this parameter `tinv`, and then talk about `inv(tinv)`? I guess the alternative is to just call it `tform`. I recognize that this has the problem of having `tform` mean different things for `WarpedView` and `InvWarpedView`.